libcaf_core/caf/hash/fnv.hpp

```diff
@@ -144,7 +144,7 @@ class fnv : public save_inspector_base<fnv<T>> {
   /// Convenience function for computing an FNV1a hash value for given
   /// arguments in one shot.
   template <class... Ts>
-  static T compute(Ts&&... xs) noexcept {
+  static T compute(Ts&&... xs) {
```

Comment:
What was clang-tidy complaining about? Can't we just make more things noexcept? I'd like to keep this `noexcept`, because there this actually cannot throw.

Reply:
Looking at the code of this function, `f.apply` is not `noexcept` so this is warning us about `bugprone-exception-escape`. This check probably generates the most warnings in the whole code base.

Reply:
Returning noexcept.